README.md

```diff
@@ -92,6 +92,10 @@ The milestone needs testing on all platforms. Please report issues when found.
     - D3 Open Source compiles level scripts in their own hogfiles. Make sure you copy and overwrite `d3-{platform}.hog`.
 
 ## Building
+#### Prerequisites
+1. [**vcpkg**](https://github.com/microsoft/vcpkg)
+    This project depends on [`vcpkg`](https://github.com/microsoft/vcpkg), a cross-platform dependency-management system developed by Microsoft. You will need to install it locally by cloning https://github.com/microsoft/vcpkg and setting your `VCPKG_ROOT` env var to the repository location.
+
```

Comment:
This kind of breaks the order of the build instructions. In order to clone the `vcpkg` repo, you need to either have the `git` command or something like [GitHub Desktop](https://desktop.github.com), but we don’t tell users to install Git until later on in the instructions.

Additionally, this step is redundant on Windows. [When you open an “x86 Native Tools Command Prompt”, it will set `VCPKG_ROOT` for you.](https://github.com/DescentDevelopers/Descent3/pull/413#discussion_r1624974614)

Also, I have a few minor critiques of this part:

- Having a list with a single item on it looks weird. I would remove the `1. `.
- The bold `vcpkg` link is redundant. I would remove the `[**vcpkg**](https://github.com/microsoft/vcpkg)`.

Reply:
Updated! I'm thinking that, in a separate PR, I'll offer up a way to make `vcpkg` a submodule of the main git repo - that would eliminate the need for a prereq setup step in the first place. For this PR however I think just the doc update should be sufficient; don't wanna scope-creep.

Reply:
OK. I think that it’s good enough at the moment. We can wait until the submodule PR to fix the build instructions order.

(Again, I would mark this conversation as resolved, but I can’t.)